src/psyclone/transformations.py

```diff
@@ -546,10 +546,21 @@ def apply(self, node, options=None):
 
         '''
         self.validate(node, options)
-        for child in node.children:
+
+        if isinstance(node, Kern):
```

Comment:
Please amend the docstring to say that `node` can also be a `Kern`. Ditto for `validate`.

Reply:
Done